File: core/src/main/java/foi/cverglici/core/data/model/DishDetailsResponse.kt
package foi.cverglici.core.data.model

data class DishDetailsResponse(
    val dishId: Int,
    val title: String,
    val description: String,
    val price: Double,
    val calories: Int,
    val protein: Double,
    val fat: Double,
    val carbohydrates: Double,
    val imgPath: String?,
    val ingredients: List<String>,
    val averageRating: Double?,
    val ratingsCount: Int
)

File: SmartMenza.Business/Models/Dishes/DishDetailsResponse.cs
namespace SmartMenza.Business.Models.Dishes
{
    public class DishDetailsResponse
    {
        public int DishId { get; set; }
        public string Title { get; set; } = string.Empty;
        public string? Description { get; set; }
        public decimal Price { get; set; }
        public int Calories { get; set; }
        public decimal Protein { get; set; }
        public decimal Fat { get; set; }
        public decimal Carbohydrates { get; set; }

        public decimal Fiber { get; set; }
        public string? ImgPath { get; set; }

        public List<string> Ingredients { get; set; } = new();

        public double? AverageRating { get; set; }

        public int RatingsCount { get; set; }
    }
}

